### PR TITLE
[update] MST  Troubleshooting

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Add troubleshooting information for connection termination with version mismatch error.

[KB Article](https://kb-managed.timescale.com/en/articles/4000086-connection-dies-while-trying-to-update-timescale-extension)

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: mst/troubleshooting.md [hunk 1]
```diff
@@ -64,6 +64,18 @@ by restarting your background workers.
 
 <CloudMSTRestartWorkers />
 
+## Connection terminates while trying to update Timescale extension
+
+When attempting to upgrade your timescaleDB version using `ALTER EXTENSION
```
Comment (charislam) [inline mst/troubleshooting.md:69]:
```suggestion
When attempting to upgrade your TimescaleDB version using `ALTER EXTENSION
```

**Review by charislam — APPROVED** [review]
Looks good, Rajie! Do you mind adding a 'new-style' troubleshooting entry as well?

🤞🏼 Hoping to get rid of this double troubleshooting thing very soon, just need to do one more teensy refactor on the page creation process and test the hell out of it!

**Comment by Rajakavitha1** [comment]
> Looks good, Rajie! Do you mind adding a 'new-style' troubleshooting entry as well?
> 
> 🤞🏼 Hoping to get rid of this double troubleshooting thing very soon, just need to do one more teensy refactor on the page creation process and test the hell out of it!

Sure!!! I will add a new-style entry for all the TS heading under MST in a new PR is that okay?

**Comment by charislam** [comment]
> > Looks good, Rajie! Do you mind adding a 'new-style' troubleshooting entry as well?
> > 🤞🏼 Hoping to get rid of this double troubleshooting thing very soon, just need to do one more teensy refactor on the page creation process and test the hell out of it!
> 
> Sure!!! I will add a new-style entry for all the TS heading under MST in a new PR is that okay?

Yep perfect, thank you!

**Review by jfjoly — APPROVED** [review]
LGTM, thanks!

**Review by Loquacity — APPROVED** [review]

**Pull request merged**